File: lib-model/src/main/java/com/minesec/msa/client/sdk/payment/model/DownloadParamResponseDto.java
package com.minesec.msa.client.sdk.payment.model;

import com.minesec.msa.client.sdk.ResultDto;

import java.io.Serializable;

import lombok.Getter;

/**
 * @author eric.song
 * @since 2022/8/26 11:33
 */
@Getter
public class DownloadParamResponseDto extends ResultDto implements Serializable {
    private DownloadParamDto data;
}

File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model_k/DownloadParamResponseDto_.kt
package com.minesec.msa.client.sdk.management.model_k

import java.io.Serializable

/**
 * @author eric.song
 * @since 2024/6/19 15:21
 */
data class DownloadParamResponseDto_(val data: DownloadParamDto_, val success: Int, val message: String): ResultDto_(success, message),Serializable
